_data/tools.yml

```diff
@@ -502,6 +502,14 @@
   v2: true
   v3: true
 
+- name: committee
+  category: schema-validators
```

Comment:
```suggestion
  category: data-validators
```

This is actually validating real data against the schemas, not validating the schemas themselves. :)

Reply:
Oh, certainly that's right!